Deploy/stacks/db/fileserver/README.md

```diff
@@ -33,6 +33,42 @@ To deploy the docker image, plese follow below steps:
 
 You should now be able to access the fileserver with a username `fs_user` and the credential you set.
 
+## Integration with the Stack Manager
+
+The fileserver can also be integrated with the stack manager. Please refer to the general stack-manager documentation for more information, especially the "Specifying a custom container" section. ([Link to stack manager README](https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-manager/README.md))  
```

Comment:
It would be preferable if the link to the specified section in the stack manager README was used, i.e. https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-manager/README.md#specifying-custom-containers